Story.md

```diff
@@ -1,38 +1,92 @@
+from the clutches of the evil Nolok.
 
-Tux and Penny were out having a nice picnic on the ice fields of Antarctica. Suddenly, a creature jumped from behind an ice bush, there was a flash, and Tux fell asleep!
+It all started with a nice picnic between the two on the ice fields of Icy Island. Suddenly,
+Nolok jumped from behind a nearby ice bush and summoned a bright flash with his mighty staff.
+The flash blinded Tux and Penny was captured in the process, leaving only a small note behind.
 
-When Tux wakes up, he finds that Penny is missing. Where she lay before now lies a letter. “Tux, my arch enemy!” says the letter. “I have captured your beautiful Penny and have taken her to my fortress. The path to my fortress is littered with my minions. Give up on the thought of trying to reclaim her, you haven't a chance! -Nolok”
+*"Tux, I have kidnapped your beloved Penny and have taken her to my fortress. The path to my
+fortress is littered with my minions. You don't have a chance of saving her!"* - Signed, Nolok.
 
-Tux looks and see Nolok's fortress in the distance. Determined to save his beloved Penny, he begins his journey.
+Realizing the trouble Penny could be in, Tux became determined to save her. Thus beginning his
+quest throughout Icy Island.
 
-![](images/Introcutscene.png "Introcutscene.png")
+Icy Island
+----------
 
-Part II
--------
+During his journey, Tux must traverse through a frozen cavern in order to reach the other side of
+the island. Upon reaching the exit of the cave, Tux noticed a strange sound in the distance. A
+yeti appeared to be watching him from afar. Before Tux could do anything, however, the yeti ran away.
+Tux was concerned by the yeti's sudden retreat but decided to continue his journey as he had spotted
+a castle.
 
-As Tux runs from the Yeti monster, he sees a ship. The monster is gaining, and the ship is leaving. Tux only has one chance... he gets on his belly and slides as fast as he can. Tux gets massive “Air Time” as he flies off the iceberg, Yeti in persuit. He screames “I'm coming, Penny!” as he lands on the ship with his belly. As he turns around, he sees the Yeti tumbling down in to the ocean...
+Tux finally reached the castle but was shocked upon seeing an empty throne room. No Nolok, no Penny.
+Only a note written by Penny. Nolok had tricked him and took off with Penny to a far forest. Tux began
+to worry even more now. He had no idea where the forest was or how to get there. His worries were cut
+short however, as out of a sudden the yeti had returned running right towards him shouting aggressively!
+Tux immediately ran towards a nearby door leading to a balcony and jumped. The yeti followed him outside
+hunting him down to a small incline, leaving Tux no other choice than to fight the yeti.
 
-Proposed story for future worlds
---------------------------------
+At last, the yeti was beaten. Rubbing his head and angrily grunting the yeti communicated his disbelief.
```

Comment:
"abrading their capitulum and snorting in incredulity, the yeti communicated his disbelief"